### ISSUE TITLE
Uncaught TypeError: path.basename is not a function

### ISSUE THREAD

**Issue by ejmurray** [atom/find-and-replace#955]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.19.5 x64
**Electron**: 1.6.9
**OS**: Microsoft Windows 7 Professional 
**Thrown From**: [find-and-replace](https://github.com/atom/find-and-replace) package 0.208.3

### Stack Trace

Uncaught TypeError: path.basename is not a function

```
At C:\ProgramData\Ernest\atom\app-1.19.5\resources\app\node_modules\find-and-replace\lib\project\results-view.js:437

TypeError: path.basename is not a function
    at ResultsView.copyPath (C:/ProgramData/Ernest/atom/app-1.19.5/resources/app/node_modules/find-and-replace/lib/project/results-view.js:437:43)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:/ProgramData/Ernest/atom/app-1.19.5/resources/app/src/command-registry.js:265:35)
    at CommandRegistry.handleCommandEvent (C:/ProgramData/Ernest/atom/app-1.19.5/resources/app/src/command-registry.js:3:65)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (C:/ProgramData/Ernest/atom/app-1.19.5/resources/app/src/command-registry.js:166:25)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (C:/ProgramData/Ernest/atom/app-1.19.5/resources/app/src/atom-environment.js:1344:34)
    at EventEmitter.outerCallback (C:/ProgramData/Ernest/atom/app-1.19.5/resources/app/src/application-delegate.js:347:31)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
  3x -6:58.7.0 core:backspace (input.hidden-input)
     -6:03.8.0 core:copy (input.hidden-input)
     -3:27.7.0 tree-view:reveal-active-file (span.hard-tab)
     -3:19.4.0 tree-view:show-in-file-manager (span.name.icon.icon-file-directory)
 59x -2:28.5.0 intentions:highlight (input.hidden-input)
     -2:26.1.0 core:copy (input.hidden-input)
 12x -2:23.1.0 intentions:highlight (input.hidden-input)
     -2:22.3.0 core:paste (input.hidden-input)
     -2:21 core:save (input.hidden-input)
  2x -0:27.2.0 find-and-replace:copy-path (span.path-name.bright)
```

### Non-Core Packages

```
atom-beautify 0.30.5 
atom-clock 0.1.15 
atom-fuzzy-grep 0.16.0 
atom-python-run 0.8.1 
autocomplete-clang 0.11.3 
autocomplete-python 1.10.2 
build 0.68.0 
busy 0.7.0 
file-icons 2.1.11 
googlesearch 1.5.0 
highlight-bad-chars 1.0.7 
highlight-selected 0.13.1 
Hydrogen 1.20.0 
hydrogen-launcher 1.2.1 
intentions 1.1.5 
language-ini 1.19.0 
language-sql-mysql 0.3.1 
linter 2.2.0 
linter-flake8 2.2.1 
linter-gcc 0.7.1 
linter-ui-default 1.6.8 
minimap 4.29.6 
minimap-highlight-selected 4.6.1 
platformio-ide 2.0.0-beta.8 
platformio-ide-debugger 1.2.4 
platformio-ide-terminal 2.6.0 
python-autopep8 0.1.3 
python-indent 1.1.0 
python-isort 0.0.7 
script 3.15.0 
scroll-searcher 5.2.0 
seti-icons 1.5.4 
tablr 1.8.3 
tool-bar 1.1.0 
touch-type-teacher 0.1.0 
```

**Comment by rsese** [comment]
Thanks for the report!  Can you confirm a few things for us?

* what were you doing at the time of the error?
* is the error is reproducible?
* if reproducible, do you see the error in safe mode (atom --safe)?

**Comment by 50Wliu** [comment]
Fixed in I believe the Atom 1.20 betas.
